apps/dashboard/src/components/workflow-editor/steps/email/maily.tsx

```diff
@@ -137,9 +137,36 @@ export const Maily = ({ value, onChange, className, ...rest }: MailyProps) => {
     ];
   }, [calculateVariables]);
 
+  /*
+   * Override Maily tippy box styles as a temporary solution.
+   * Note: These styles affect both the bubble menu and block manipulation buttons (drag & drop, add).
+   * TODO: Request Maily to expose these components or provide specific CSS selectors for individual targeting.
+   */
+  const overrideTippyBoxStyles = () => (
+    <style>
+      {`
+          .tippy-box {
+            padding-right: 20px;
+            pointer-events: auto;
+
+            .mly-cursor-grab {
+              background-color: #fff;
+              border-radius: 4px;
+              box-shadow: 0px 0px 2px 0px rgba(0, 0, 0, 0.04), 0px 1px 2px 0px rgba(0, 0, 0, 0.02);
+              border-radius: 4px;
+            }
+          }
+        `}
+    </style>
+  );
+
```

Comment:
I'll open a PR to maily to add more specific selectors so we can target the elements individually. That would also enable us to hide them completely and create or own.

Otherwise Maily would need to expose the bubble menu which I think is quite a lot of work.